services/new_mac_setup.md

```diff
@@ -0,0 +1,60 @@
+# New Mac Setup
+This is a get started guide to make it easier to setup your mac.
+
+When you get a new mac here are some helpful guides to setting it up for software development in RDSS.
```

Comment:
Remove reference to RDSS, since we're putting this into the general ITIMS handbook.